src/Automation/Interfaces/IScanResultsAssembler.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Axe.Windows.Core.Bases;
+
+namespace Axe.Windows.Automation.Interfaces
+{
+    /// <summary>
+    /// Provides methods used to assemble a <see cref="ScanResults"/> object
+    /// </summary>
+    public interface IScanResultsAssembler
```

Comment:
This should be internal instead of public. No one outside the assembly needs to know about it.

Reply:
Now internal